src/NewDataList.js

```diff
@@ -209,6 +210,34 @@ module.exports = kind(
 		}
 		this.positionChildren();
 	},
+
+	/**
+	* @private
+	*/
+	refreshThresholds: function () {
+		var tt = this.threshold;
+
+		if (tt) {
+			var
+				v = (this.direction === 'vertical'),
+				val = v ? this.scrollTop : this.scrollLeft,
+				delta = this.delta,
+				cb = this.cachedBounds ? this.cachedBounds : this._getScrollBounds(),
```

Comment:
@KunmyonChoi, at a glance, I don't think so. I think it would be relatively straightforward to make it happen, though -- we would just override `showingChangedHandler()` and call `refreshThresholds()` before calling the super method. If you write up a JIRA demonstrating a problem case, we will take a look.